.eslintrc.js

```diff
@@ -356,6 +356,11 @@ module.exports = {
                 ],
                 allowSameFolder: true,
               },
+              {
+                target: ['src/core/**/*'],
+                from: ['plugins/**/*', 'src/plugins/**/*'],
```

Comment:
Should we add legacy plugin paths too?

Reply:
We already have the rule to prevent importing legacy code from the new platform.
https://github.com/elastic/kibana/pull/51563/files#diff-e4403a877d80de653400d88d85e4801aR322-R358
I will add this check anyway to avoid confusion.